docs/vm/live-migration.md

```diff
@@ -15,27 +15,96 @@ description: Live migration means moving a virtual machine to a different host w
   <link rel="canonical" href="https://docs.harvesterhci.io/v1.5/vm/live-migration"/>
 </head>
 
-Live migration means moving a virtual machine to a different host without downtime.
+Live migration means moving a virtual machine to a different host without downtime. A couple of comprehensive processes and tasks are done under the hood to fulfill the live migration.
+
+## Prerequisites
+
+Live migration can occur when the following requirements are met:
+
+- The cluster has at least one schedulable node (in addition to the current node) that matches all of the virtual machine's scheduling rules.
+
+- The migration target node has enough available resources to host the virtual machine.
+
+- The CPU, memory, [volumes](./create-vm.md#volumes), devices and other resources requested by the virtual machine can be copied or rebuilt on the migration target node while the source virtual machine is still running.
+
+## Non-migratable Virtual Machines
+
+A virtual machine is considered non-migratable if it has one or more of the following:
+
+- Volume with the following properties:
+
+    - Type: `CD-ROM` or `Container Disk`
+    - Access mode: `ReadWriteOnce`
+    - StorageClass replica count: `1` (This is not detected in all cases.)
+
+- Host devices passthrough such as `PCI` and `vGPU`
+
+- [Node selector](./create-vm.md#node-scheduling) that binds the virtual machine to a specific node
+
+- [Scheduling rules](./create-vm.md#node-scheduling) that match only one node
+
+  The following are examples of rule conditions that are checked at runtime. For more information, see [Automatically Applied Affinity Rules](./create-vm.md#related-networking-concepts).
+
+    - The virtual machine is attached to a cluster network that covers only one node.
+
+    - CPU pinning is enabled on the virtual machine, and CPU Manager is only enabled on one node.
+
 
 :::note
 
-- Live migration is not allowed when the virtual machine is using a management network of bridge interface type.
-- Live migration is not allowed when the virtual machine has any volume of the `CD-ROM` type. Such volumes should be ejected before live migration.
-- Live migration is not allowed when the virtual machine has any volume of the `Container Disk` type. Such volumes should be removed before live migration.
-- Live migration is not allowed when the virtual machine has any `PCIDevice` passthrough enabled. Such devices need to be removed before live migration.
-- Live migration is not allowed when the volumeAccessMode of any volume in the virtual machine is `ReadWriteOnce`. Such volumes should be removed before live migration.
+To live-migrate the virtual machine, you must first remove non-migratable devices and add schedulable nodes.
 
 :::
 
+## Live-migratable Virtual Machines
+
+Besides the `non-migratable virtual machines`, the rest of the running virtual machines are considered `live-migratable`.
+
 ## How Migration Works
 
+### The `VirtualMachineInstanceMigration` Object
```

Comment:
I feel like this is getting into the weed. What should the reader do with this info?

Can we just add a link to reference the comprehensive KB on live migration at https://harvesterhci.io/kb/vm_live_migration_policy_and_configuration? Feels like the KB might be a better home for these  details.

Reply:
The KB was written around two years ago, and now many changes have been introduced:

Here the `second-level` title is `How Migration Works`, and it just mentions `cpu model matching`, adding the `the vmim object` expands it.

There are 4 ways to trigger migration on v1.6.0:  **Migrate** menu, node maintenance, upgrade, hot-plug, and each has some characters. 

Both of doc && kb need updates to describe things from different perspective, and cross-refer.
 
 Though some document setcions seem redundant, but it is also true that not every user has the patience to read all.
 
Certain degree of in-depth description of `live migration` on `live-migration.md` does not harm, at least.